### ISSUE TITLE
[Feature] Improvements to voiceover on journey details

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#551]
## Origin
_Links to received feedback, user research or other findings.

Discovered while QA-testing for beta in voiceover mode:
Nearby -> Departure details:
- "Mellomstopp" is missing info about button. Now it reads "10 mellomstopp". 
- Each line with time and leg name is now separate focus frames. Suggestion that each line (for instance time + leg is read together. Makes it easier to move between the different legs of the journey as you only have to move the finger up/down, instead of up, down, left, right.

## Motivation
_A short description of what user needs or business goals this feature will solve._

## Hypotheses and assumptions
_A list of hypotheses and assumptions we have made about the user or the proposed solution._

## Proposed solution
_A coarse description of a proposed solution, that may include wireframes and graphic design._
- "Mellomstopp" must be a touchable element, so it's accessible to the screen reader, and the correct instruction is read out ("10 mellomstopp, knapp, dobbelklikk for å aktivere")
- Each leg should indeed be one accessible element. Consider doing this change along with implementing the new design (as shown below, please link this issue to the correct issue if and when it's made)
![Journey](https://user-images.githubusercontent.com/61825573/95854269-fee98f00-0d56-11eb-80a9-2f949ff85fa1.png)
